plutus-ledger-constraints/src/Ledger/Constraints/OffChain.hs

```diff
@@ -392,14 +392,12 @@ mkTx lookups txc = mkSomeTx [SomeLookupsAndConstraints lookups txc]
 
 -- | Each transaction output should contain a minimum amount of Ada (this is a
 -- restriction on the real Cardano network).
---
--- TODO: In the future, the minimum Ada value should be configurable.
-adjustUnbalancedTx :: UnbalancedTx -> UnbalancedTx
-adjustUnbalancedTx = over (tx . Tx.outputs . traverse) adjustTxOut
+adjustUnbalancedTx :: Ledger.Ada -> UnbalancedTx -> UnbalancedTx
+adjustUnbalancedTx minAdaTxOut = over (tx . Tx.outputs . traverse) adjustTxOut
```

Comment:
The parameter should be `coinsPerUtxoWord` instead of `minAdaTxOut`, no? That's what you're passing in through the handlers. In that case, this implementation should be different.